src/components/panel/select-view/Views.tsx

```diff
@@ -13,9 +15,10 @@ interface SelectViewButtonsProps {
 const Views: FC<SelectViewButtonsProps> = ({ updateSelectedButton }) => {
   const [selectedView, setSelectedView] = useState(useUIStore.getState().defaultView)
   const { t } = useTranslation()
+  const views = useConfigStore(state => state.config.views)
 
-  const buttonsData = {
-    pip: {
+  const defaultButtonsData = {
```

Comment:
For `buttonData` I would drop the object structure and turn it into an array, if we now want to rely on the order.